courses/static/js/compare-selector.js

```diff
@@ -106,7 +106,7 @@
 
         handleCourseAddition: function() {
             //Don't allow more than 10
-            if (this.selectedCourses.length === 10) {
+            if (this.selectedCourses.length === 25) {
```

Comment:
number, rather than a comment can you make this MAX_ALLOWED_COURSES or something that makes sense